tavern/app.go

```diff
@@ -122,12 +145,25 @@ func NewServer(ctx context.Context, options ...func(*Config)) (*Server, error) {
 	router.Handle("/cdn/", cdn.NewDownloadHandler(client))
 	router.Handle("/cdn/upload", cdn.NewUploadHandler(client))
 
+	// Log Middleware
+	httpLogger := log.New(os.Stderr, "[HTTP] ", log.Flags())
+	handlerWithLogging := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		authName := "unknown"
+		id := auth.IdentityFromContext(r.Context())
+		if id != nil {
+			authName = id.String()
+		}
+
+		httpLogger.Printf("%s (%s) %s %s\n", r.RemoteAddr, authName, r.Method, r.URL)
```

Comment:
Is r.Method the graphql mutation / query name? Eg. Queue task?

Reply:
> may want more verbosity what systems commands were queued on

Honestly probably just used for debugging purposes, but you do get that as long as you provide variables instead of hardcoding values in the queries.